packages/playwright-core/src/utils/registry.ts

```diff
@@ -738,7 +739,7 @@ export function buildPlaywrightCLICommand(sdkLanguage: string, parameters: strin
     case 'csharp':
       return `pwsh bin\\Debug\\netX\\playwright.ps1 ${parameters}`;
     default:
-      return `npx playwright ${parameters}`;
+      return `npx playwright@${packageJSON.version} ${parameters}`;
```

Comment:
The version is not always necessary, but depending on where you call this from (and when) it can be.

Reply:
use `getPlaywrightVersion` instead!

Reply:
Updated! Let me know if it's good to merge!